gstd/src/common/handlers.rs

```diff
@@ -24,29 +24,26 @@
 //! Panic handlers are available in two implementations -
 //! debug and non-debug mode, for programs built in `wasm32` architecture.
 //! For `debug` mode it provides more extensive logging.
+#![cfg(target_arch = "wasm32")]
```

Comment:
I'd move `target_arch="wasm32"` to `mod.rs`
https://github.com/gear-tech/gear/blob/3d0d23f85ec3528015e8ae2a0e534b0283ea6a36/gstd/src/common/mod.rs#L22-L23

Reply:
Fixed in 13e8c07